### ISSUE TITLE
Syntax error with placeholders as value

### ISSUE THREAD

**Issue by Grmiade** [Microsoft/typescript-styled-plugin#39]
#### Steps to reproduce
1. Type this code
```typescript
const navBarWidth = 15

interface ContainerProps {
  withNavBar?: boolean
}

const Container = styled.div`
  flex: 1;

  & > ${Content} {
    margin-left: ${({ withNavBar }: ContainerProps) => (withNavBar ? `${navBarWidth}em` : 0)};
  }

  & > ${StyledNavBar} {
    margin-left: ${({ withNavBar }: ContainerProps) =>
      withNavBar ? 0 : `-${navBarWidth}em`} !important;
  }
`
```
#### Expected
No error

#### Actual
The `property value expected` error is raised.

<img width="857" alt="capture d ecran 2018-02-20 a 14 30 21" src="https://user-images.githubusercontent.com/11244106/36426569-aa7dd7f6-164a-11e8-8226-95a77d3fb805.png">

**Comment by rzec-r7** [comment]
I am also getting a similar error even though the code compiles and generates the correct css:

![screen shot 2018-02-27 at 7 45 55 am](https://user-images.githubusercontent.com/30290479/36729503-675eae80-1b92-11e8-9859-d69a8c001431.png)
